### ISSUE TITLE
Having trouble with one-click login

### ISSUE THREAD

**Issue by bhealz** [jakcodex/muledump#255]
I am migrating from one computer to another (new one has Windows 10).  I installed latest firefox, flash player, addons, auto it, muledump, etc. I am using the same accounts.js and same autoit script (changed file locations for flash projector). On the new computer, it is successfully opening up my flash projector but not logging me on my account. I tried using realmeye.com/appspot, rotmg main website, AGC swf, and the AGC swf converted to an executable to open the flash projector (with and without admin privileges). None of them are logging into my accounts. Maybe the computer can't correctly find the pathway to the stored cache/options? I would appreciate it if someone can help me out.

login script:
https://pastebin.com/8u4mLW3G

**Comment by jakcodex** [comment]
Deca's X27 ROTMG release introduced breaking changes to OCL which require you update your OCL script to the latest version.

The specific breaking change is that the valid hostname for accessing the game files is now https://www.realmofthemadgod.com/. The URL for the latest swf is https://www.realmofthemadgod.com/client.

Download the latest OCL script, update it with your preferences and using the new swf url for flash proj, and then rerun the script choosing the reinstall option. Should start working for you.

On a side note: Are you using an accounts.js file out of preference, or are you unaware of SetupTools? Your feedback helps me better understand the use case for users who still use the legacy accounts.js file. Thanks!

**Comment by bhealz** [comment]
Still not working and everything is up to date, even used the new swf URL you mentioned.

I am just using an account.js file out of preference since I'm not really comfortable with the SetupTools.

**Comment by jakcodex** [comment]
Can you share your updated OCL script? And to confirm, you reran the script and chose reinstall?

**Comment by bhealz** [comment]
https://pastebin.com/zvDzUJ4g

Yes, I reran the script with reinstallation, restarted the comp, etc.
I think it's that it's unable to update the account cache properly.

**Comment by jakcodex** [comment]
Have you tried specifically setting

```js
ShellExecute("https://www.realmofthemadgod.com/client")
```

instead of

```js
ShellExecute("https://www.realmofthemadgod.com/")
```

On a side note, there is a known bug with OCL not updating rotmg.sol for certain users, but every single reported case has been specifically for browser-based OCL and not Flash projector. If this is the same bug affecting you, it'd been ridiculously hard to get meaningful test info to debug.

**Comment by jakcodex** [comment]
Hi there,

We're testing an updated OCL script that fixes path issues that affected some users. If you are interested, you can try the updated script to see if it solves this issue for you.

https://raw.githubusercontent.com/jakcodex/muledump/dev/lib/mulelogin.au3

**Comment by jakcodex** [comment]
Pretty sure this is fixed in v9.6 which went out today. Cheers.
